packaging/installer/methods/kickstart.md

```diff
@@ -11,92 +11,133 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/instal
-- `--install`: Specify an alternative install prefix. 
-- `--disable-cloud`: For local builds, don’t build any of the cloud code at all. For native packages and static builds, 
+- `--reinstall-even-if-unsafe`: Even try to reinstall if we don't think we can do so safely (implies `--reinstall`).
+- `--disable-cloud`: For local builds, don’t build any of the cloud code at all. For native packages and static builds,
     use runtime configuration to disable cloud support.
-- `--auto-update-type`: Specify how auto-updates are to be scheduled, overriding auto-detection. 
-- `--disable-telemetry`: Disable anonymous statistics. 
-- `--local-files`: Used for [offline installations](offline.md). Pass four file paths: the Netdata
-    tarball, the checksum file, the go.d plugin tarball, and the go.d plugin config tarball, to force kickstart run the
-    process using those files. This option conflicts with the `--stable-channel` option. If you set this _and_
-    `--stable-channel`, Netdata will use the local files.
--    `all`: Ask for all dependencies in the dependency handling script. 
-     
- > Note: The `all` and `--local-files` parameters are scheduled to be removed in a forthcoming update. 
- 
+- `--require-cloud`: Only install if Netdata Cloud can be enabled. Overrides `--disable-cloud`.
+- `--install`: Specify an installation prefix for local builds (by default, we use a sane prefix based on the type of system).
+
+Additionally, the following environment variables may be used to further customize how the script runs (most users
+should not need to use special values for any of these):
+
+- `TMPDIR`: Used to specify where to put temporary files. On most systems, the default we select automatically
+  should be fine. The user running the script needs to both be able to write files to the temporary directory,
+  and run files from that location.
+- `ROOTCMD`: Used to specify a command to use to run another command with root privileges if needed. By default
+  we try to use sudo, doas, or pkexec (in that order of preference), but if you need special options for one of
+  those to work, or have a different tool to do the same thing on your system, you can specify it here.
+- `DO_NOT_TRACK`: If set to a value other than 0, behave as if `--disable-telemetry` was specified.
+- `NETDATA_INSTALLER_OPTIONS`: Specifies extra options to pass to the static installer or local build script.
+
 ### Connect node to Netdata Cloud during installation
 
 The `kickstart.sh` script accepts additional parameters to automatically [connect](/claim/README.md) your node to Netdata
 Cloud immediately after installation. Find the `token` and `rooms` strings by [signing in to Netdata
 Cloud](https://app.netdata.cloud/sign-in?cloudRoute=/spaces), then clicking on **Connect Nodes** in the [Spaces management
 area](https://learn.netdata.cloud/docs/cloud/spaces#manage-spaces).
```

Comment:
```suggestion
The `kickstart.sh` script accepts additional parameters to automatically [connect](/claim/README.md) your node to Netdata Cloud immediately after installation. 

> Note: You either need to run the command with root privileges or run it with the user that is running the agent.  More details: [Connect an agent without root privileges](/claim/README.md#connect-an-agent-without-root-privileges) section.

To automatically claim nodes after installation: 
1. Sign in to [Netdata Cloud](https://app.netdata.cloud/sign-in?cloudRoute=/spaces)
2. Go to the [Spaces management area](https://learn.netdata.cloud/docs/cloud/spaces#manage-spaces)
3. Click on **Connect Nodes**
4. Find the `token` and `rooms` strings and specify your nodes:
```
@Ferroin I untangled the paragraph that contained steps into an ordered list for clarity. Please check if the wording and workflow makes sense.